### DISCUSSION TITLE
Is it possible to configure 2 cells on one B210?

### DISCUSSION THREAD

**Discussion by yc541** [srsran/srsRAN_Project#820] [General Help] [answered]
I have been trying to make a handover scenario on one USRP since the experiment with 2 B205mini wasn't completely successful https://github.com/srsran/srsRAN_Project/discussions/803#discussioncomment-10657345

Making two cells on one X310 was successful and there are UE HOs according to RSRP measurements. 

I was trying to do the same on one B210 but the UHD wasn't happy with the configuration:

_Error:  failed to create transmit stream 1. RuntimeError: b200: 2 RX 1 TX and 1 RX 2 TX configurations not possible._

The terminal logs, gnb logs and config file are attached. Please let me know if anything can be tweaked. Many thanks.

[B210_logs.zip](https://github.com/user-attachments/files/17043221/B210_logs.zip)

**Comment by dvdgrgrtt** [comment]
Hi!
As mentioned [here](https://docs.srsran.com/projects/project/en/latest/tutorials/source/handover/source/index.html#x310-usrp), the B210 doesn't fulfill the requirements needed for running 2 cells - sorry.

**Reply by yc541** [reply, marked as answer]
Ah, I missed that bit in the guide document. Thanks for pointing that out.

**Comment by Zplusless** [comment]
Hi @dvdgrgrtt, I have reviewed the tutorial you referenced.  Is it possible to set up two cells in zmq mode, if I start one CU and two DU following [this tutorial](https://docs.srsran.com/projects/project/en/latest/tutorials/source/cu_du_split/source/index.html)?

By the way, if my understanding is correct, srsRAN does not support one DU with multiple cells, so multiple DUs need to be deployed. right？

**Reply by dvdgrgrtt** [reply]
Hi!
The tutorial you want to look at is [this one](https://docs.srsran.com/projects/project/en/latest/tutorials/source/handover/source/index.html) - or you can mix the two if you want to run split CU and DUs. If you want to run it with ZMQ, you need some extra machinery to mix the two signals - have a look at this other old [tutorial](https://docs.srsran.com/projects/4g/en/latest/app_notes/source/handover/source/index.html#introduction), the main idea still holds.

And yes, you need to deploy one DU per cell.

**Reply by Zplusless** [reply]
Thanks！ It really helps!
